frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -1069,9 +1097,32 @@ export const funnelLogic = kea<funnelLogicType>({
             actions.loadEventWithPropertyCorrelations(eventName)
         },
 
-        excludeProperty: () => {
+        excludeProperty: ({ propertyName }) => {
+            const oldCurrentTeam = teamLogic.values.currentTeam
+
+            if (!oldCurrentTeam) {
+                return
+            }
+
+            const oldCorrelationConfig = oldCurrentTeam.correlation_config
+
+            const excludedPropertyNames = (oldCorrelationConfig?.excluded_person_property_names || []).concat([
```

Comment:
worth checking if it's not already in `excluded_person_property_names` ? in which case we don't update the team?